src/sentry/sentry_apps/api/bases/sentryapps.py

```diff
@@ -49,6 +49,17 @@ def wrapped(self, *args, **kwargs):
     return wrapped
 
 
+def _handle_sentry_app_exception(exception: Exception):
```

Comment:
nit: since this only exists in 1 place now you can also just move this into `IntegrationPlatformEndpoint`